pkg/vulnsrc/vulnerability/vulnerability.go

```diff
@@ -44,6 +44,27 @@ func getSeverity(details map[string]types.VulnerabilityDetail) types.Severity {
 	return types.SeverityUnknown
 }
 
+func getVendorSeverity(details map[string]types.VulnerabilityDetail) types.VendorSeverity {
+	vs := make(types.VendorSeverity)
+	for _, source := range sources {
+		switch d, ok := details[source]; {
+		case !ok:
+			continue
+		//case d.CvssScore > 0:
+		//	return scoreToSeverity(d.CvssScore)
+		//case d.CvssScoreV3 > 0:
+		//	return scoreToSeverity(d.CvssScoreV3)
+		//case d.Severity != 0:
+		//	return d.Severity
+		//case d.SeverityV3 != 0:
+		//	return d.SeverityV3
```

Comment:
The vendors sometimes don't have their own severity yet, especially immediately after the vulnerability is found. To support the case, we should store NVD severity as well. Of course, sometimes NVD doesn't have the severity, while the vendor has.

Reply:
Are you suggesting we need both `Severity` and `VendorSeverity`?